recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -16,3 +16,7 @@ do_install:append() {
 }
 
 ALTERNATIVE_PRIORITY[resolv-conf] = "10"
+
+# Notice: firewall from networkmanager is not used at all, so nftables or iptables settings are not needed.
+# To remove nftables and python from the root filesystem, the network manager firewall has been disabled.
+NETWORKMANAGER_FIREWALL_DEFAULT = ""
```

Comment:
- "network manager"  -> "networkmanager"/"NetworkManager"
- the networkmanager is not used to configure the firewall ... there is no explicit "network manager firewall".

Reply:
comment refactored